### PR TITLE
JBIDE-25308 - minishift_home is not passed to launch command

### PR DESCRIPTION
**Pull request by robstryker**
Signed-off-by: Rob Stryker <rob@oxbeef.net>

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by robstryker** [comment]
testPR

File: plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/adapter/controllers/CDK3LaunchController.java [hunk 1]
```diff
@@ -98,12 +98,23 @@ static String getProfileString(IServer server) {
 		}
 		return profiles;
 	}
+	
+	protected IServer getServerFromLaunch(ILaunchConfigurationWorkingCopy wc) throws CoreException {
```
Comment (adietish) [inline plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/adapter/controllers/CDK3LaunchController.java]: I dont see the benefit of this additional method.

**Comment by adietish** [comment]
@robstryker I let you dismiss @jeffmaury's review and merge.

File: plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/adapter/controllers/CDK3LaunchController.java (hunk 1 above)
Comment (robstryker) [reply plugins/org.jboss.tools.openshift.cdk.server/src/org/jboss/tools/openshift/cdk/server/core/internal/adapter/controllers/CDK3LaunchController.java]: I think I had originally put it there for the test to override, but it became unnecessary. Still, doesn't hurt anything.

**Pull request merged**